instrumentation/active_storage/lib/opentelemetry/instrumentation/active_storage/instrumentation.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActiveStorage
+      # The {OpenTelemetry::Instrumentation::ActiveStorage::Instrumentation} class contains logic to detect and install the ActiveStorage instrumentation
+      #
+      # Installation and configuration of this instrumentation is done within the
+      # {https://www.rubydoc.info/gems/opentelemetry-sdk/OpenTelemetry/SDK#configure-instance_method OpenTelemetry::SDK#configure}
+      # block, calling {https://www.rubydoc.info/gems/opentelemetry-sdk/OpenTelemetry%2FSDK%2FConfigurator:use use()}
+      # or {https://www.rubydoc.info/gems/opentelemetry-sdk/OpenTelemetry%2FSDK%2FConfigurator:use_all use_all()}.
+      #
+      # ## Configuration keys and options
+      #
+      # ### `:disallowed_notification_payload_keys`
+      #
```

Comment:
It looks like the default is missing on just this config:
```suggestion
      #
      # - `array` **default** `[]`
      #
```

Reply:
I agree, fixed in https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/1313/commits/613e1b6f528f6ea077e1d64f1013d04a6ebb1e8a